e2e/atlas/clusters_test.go

```diff
@@ -73,14 +68,8 @@ func TestClustersFlags(t *testing.T) {
 		)
 		cmd.Env = os.Environ()
 		resp, err := cmd.CombinedOutput()
-
-		if err != nil {
-			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
-		}
-
-		if !strings.Contains(string(resp), "Cluster available") {
-			t.Errorf("got=%#v\nwant=%#v\n", string(resp), "Cluster available at:")
-		}
+		a.NoError(err)
+		a.Contains(string(resp), "Cluster available", fmt.Sprintf("got=%#v\nwant=%#v\n", string(resp), "Cluster available at:"))
```

Comment:
[nit] no need to provide the error message
```suggestion
		a.Contains(string(resp), "Cluster available")
```